config/software/td-agent-files.rb

```diff
@@ -15,7 +15,8 @@
     project_name = project.name # for ERB
     project_name_snake = project.name.gsub('-', '_') # for variable names in ERB
     gem_dir_version = "2.6.0"
-
+    use_systemd = ohai["os"] == "linux" && ohai["platform_family"] == "suse"
+    
```

Comment:
Please remove the trailing whitespace here.

Reply:
I only saw the empty line after; thus removed

Reply:
The empty line was fine (it was there before your change, too). I was talking about the spaces you've added to that line.
Can you please re-instate it, but without the trailing spaces?

Reply:
I just checked on both console and webpage ; also removed the whole line then re-added. i don't see any white space in my end..